tinygrad/runtime/ops_metal.py

```diff
@@ -64,7 +64,6 @@ def __call__(self, global_size, local_size, *bufs, wait=False):
     if local_size is None: local_size = [32]
     local_size += [1] * (3-len(local_size))
 
-    assert prod(local_size) <= self.pipeline_state.maxTotalThreadsPerThreadgroup(), f"local size {local_size} bigger than {self.pipeline_state.maxTotalThreadsPerThreadgroup()} with exec width {self.pipeline_state.threadExecutionWidth()} memory length {self.pipeline_state.staticThreadgroupMemoryLength()}"
```

Comment:
Why was this removed?

Reply:
it was for testing, I've reverted this change

Reply:
No it's not. I wrote that assert.